src/page/Home.tsx

```diff
@@ -15,20 +17,132 @@ class Home extends React.Component<{}>
-                        <SearchBox
-                            onChange={this.onSearch}
-                            placeholder="Search variants"
-                        />
-                    </Col>
-                </Row>
-                <Row className="py-4">
-                    <Col className="m-auto">
-                        {this.inputText}
-                    </Col>
-                </Row>
+            <div>
+                <div className="text-center">
+                    <Row>
+                        <Col lg="8" className="home-logo">
+                            <GenomeNexusLogo imageHeight={130}/> 
+                                GenomeNexus
+                        </Col>
+                    </Row>
+                    <Row>
+                        <Col lg="8" className="home-description">
+                            A resource for annotation and interpretation of genetic variants
+                        </Col>
+                    </Row>
+                    <Row>
+                        <Col lg="6" className="m-auto">
+                            <SearchBox
+                                onChange={this.onSearch}
+                                placeholder="Search variants"
+                                searchIconClassName="fa fa-search"
+                            />
+                        </Col>
+                    </Row>
+                    <Row>
+                        <Col lg="7" className="home-function-description">
+                            Genome Nexus will integrate information from a variety of existing resources, 
+                            including databases that convert DNA changes to protein changes, predict the 
+                            functional effects of protein mutations, and contain information about mutation 
+                            frequencies, gene function, variant effects, and clinical actionability.
+                        </Col>
+                    </Row>
+                    <Row className="justify-content-md-center">
+                        <Col lg="2">
+                            <Button onClick={() => window.open("http://genomenexus.org/swagger-ui.html", "_blank")} variant="outline-primary">Try live API</Button>
+                        </Col>
+                        <Col lg="2">
+                            <Button onClick={this.linkToExamples} variant="link">See examples</Button>
+                        </Col>
+                    </Row>
+                </div>
+
+                <div id="home-example-container" className="home-example-container">
+                    <Row>
+                        <Col lg="8" className="home-query-example-header">
+                                Query Examples
+                        </Col>
+                    </Row>
+                    <Row>
+                        <Col lg="8" className="home-query-example-table">
+                            <Table responsive>
```

Comment:
I am not really fond of using `Table` for alignment purposes, maybe use `Row` and `Col` instead?

Reply:
why `table` is not good? I think if we use `row` and `col` and make the window smaller, the second column will go under the first column, while the table will stay as two columns.

Reply:
with `Row`s and `Col`s the code looks cleaner to me. But if it is too much work to align things with `Row`s and `Col`s we can keep it this way.